packages/design-system/src/components/checkbox/index.js

```diff
@@ -78,7 +77,7 @@ const CheckboxContainer = styled.div(
         }
       }
 
-      &.${FOCUS_VISIBLE_SELECTOR}:not(:active) ~ ${Border} {
+      &:focus-visible:not(:active) ~ ${Border} {
```

Comment:
Unrelated to this line but didn't have a place to add it: search for `.focus-visible` in the repo still gives one result :)

Reply:
Good catch, thanks!